client/elements/addons/sc-static-page.html

```diff
@@ -15,28 +15,37 @@
                   currentId: {
                       type: String,
                       value: ''
+                  },
+                  language: {
+                      type: String,
+                      statePath: 'siteLanguage',
+                      observer: '_siteLanguageChanged'
+                  },
+                  fallbackLanguage: {
+                      type: String,
+                      value: 'en'
+                  },
+                  languageStringPath: {
```

Comment:
In hindsight, it would make more sense if this was named `localizedStringsPath` or `localizedStringResourcesPath`. What do you think?